src/components/Carousel/Carousel.module.css

```diff
@@ -0,0 +1,66 @@
+.carousel {
+    background-color: #212529;
+    display: flex;
+    flex-direction: column;
+    overflow: hidden;
+    align-items: center;
+}
+
+.carousel__buttons {
+    display: flex;
+    justify-content: center;
+}
+
+.carousel__buttons-container {
+    display: grid;
+    padding: min(10%, 50px) 0;
+    grid-template-columns: repeat(6, 1fr) 3fr repeat(6, 1fr);
+}
+
+.carousel__direction-buttons-container {
+    background-color: #6C757D;
+    display: flex;
+    justify-content: space-between;
+    grid-column-start: 7;
+    border-radius: 25px;
+}
+
+.carousel__eye-button {
+    grid-column-start: 9;
+}
+.carousel__download-button {
+    grid-column-start: 11;
+}
+
+.carousel__buttons-container button {
```

Comment:
I think the double underscores in the names are verbose and not very useful. `carousel-buttons-container` should be sufficient.

Reply:
So would you say for this I can just avoid using the BEM notation?

Reply:
yeah this isn't a particularly big site. Especially since we have everything separated by components it's messier to have fancy naming conventions.